.github/workflows/ci.yml

```diff
@@ -28,11 +28,17 @@ jobs:
   run-tests:
     runs-on: [self-hosted, kind-cluster]
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
+    - uses: hashicorp/setup-terraform@v2
+      with:
+        terraform_version: 0.15.1
```

Comment:
we constraint to run the plan with terraform `~>1.0`. Does it make sense to run the `terraform fmt` with older version?

Reply:
Good point.  I've bumped it.